src/sage/categories/drinfeld_modules.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is the default in python 3, the coding magic comments have not been needed for a long time now. Ditto other occurences